### ISSUE TITLE
Oracle Linux 9: Analyze vulnerability discrepancies in the report between 4.7.3 and 4.8.0 

### ISSUE THREAD

**Issue by GGP1** [wazuh/wazuh-qa#5123]
|Platform|From|To|
|--|--|--|
|Oracle Linux 9|4.7.3|4.8.0-beta4|

We are tasked with performing a comprehensive analysis of vulnerability discrepancies reported between versions 4.7.3 and 4.8.0 in Oracle Linux 9. This entails scrutinizing each vulnerability identified in the report, cross-referencing them with official sources such as the National Vulnerability Database (NVD), Canonical, Red Hat Security Advisories (RHSA), and other trusted sources.

To accomplish this, we will follow a systematic approach:

1. **Gathering Vulnerability List:** We will compile a comprehensive list of vulnerabilities detected in versions 4.7.0 and 4.8.0. This list will be sourced from the provided report and any additional channels that may provide pertinent information. **([Link](https://docs.google.com/spreadsheets/d/1I1mNNNAmNs5itV06HjFs8T6gK8DWxa-sJfPn0mwesb4/edit?usp=sharing))**

2. **Package Inventory Examination:** We will thoroughly examine the package inventory associated with the aforementioned versions to ensure accuracy in vulnerability identification and tracking.

3. **Verification with Official Sources:** Each identified vulnerability will be meticulously compared with information available from official sources such as the NVD, Canonical, RHSA, and other relevant platforms. This step is crucial in validating the existence and severity of each vulnerability.

4. **Analysis and Documentation:** We will document our findings in a structured manner, presenting a detailed analysis of each vulnerability and highlighting any variances or discrepancies encountered between the provided report and official sources.

5. **Compilation of Results:** Based on our analysis, we will populate a comprehensive table detailing the results of our comparison. This table will serve as a reference point for understanding the status of each vulnerability and any deviations observed.

6. **Recommendations:** Finally, we will provide recommendations based on our findings, including potential actions to address any identified vulnerabilities and mitigate associated risks effectively.

By adhering to this methodical approach, we aim to ensure thoroughness and accuracy in our analysis of vulnerability discrepancies between versions 4.7.0 and 4.8.0. This endeavor will contribute to enhancing the overall security posture of our system and bolstering our resilience against potential threats.

**Comment by GGP1** [comment]
# Final results

The analysis has been completed:
- **Vulnerabilities in `4.7.3`, not reported in `4.8.0` ([link](https://docs.google.com/spreadsheets/d/1I1mNNNAmNs5itV06HjFs8T6gK8DWxa-sJfPn0mwesb4/edit#gid=1862435592)):** 28
  - :green_circle: 1 correctly reported.
  - :yellow_circle: 0 doubtful.
  - :red_circle: 27 incorrectly reported.
- **Vulnerabilities in `4.8.0`, not reported in `4.7.3` ([link](https://docs.google.com/spreadsheets/d/1I1mNNNAmNs5itV06HjFs8T6gK8DWxa-sJfPn0mwesb4/edit#gid=1766057178)):** 100
  - :green_circle: 25 correctly reported.
  - :yellow_circle: 6 doubtful.
  - :red_circle: 69 incorrectly reported.
 
> [!NOTE]
> Many vulnerabilities were fixed in previous package versions but they were still reported. I suspect this may be because of the packages shipped in the [latest source packages for Oracle Linux 9](https://yum.oracle.com/repo/OracleLinux/OL9/baseos/latest/x86_64/index_src.html).

**Comment by sebasfalcone** [comment]
### CVE-2023-28321

In the analysis is stated that this vulnerability is fixed on version "7.76.1-23.el9_2.2"

But both the NVD and Oracle Linux state that every version below 8.1.0 is affected:
- https://linux.oracle.com/cve/CVE-2023-28321.html

```json
        {
          "cpes": [
            "cpe:2.3:a:haxx:curl:*:*:*:*:*:*:*:*"
          ],
          "defaultStatus": "unaffected",
          "product": "curl",
          "vendor": "haxx",
          "versions": [
            {
              "lessThan": "8.1.0",
              "status": "affected",
              "version": "0",
              "versionType": "custom"
            }
          ]
        }
```

The installed version is `7.76.1-26.el9_3.2`, so it is vulnerable
